controllers/topology/internal/mergepatch/mergepatch.go

```diff
@@ -193,5 +201,10 @@ func (h *Helper) Patch(ctx context.Context) error {
 	if !h.HasChanges() {
 		return nil
 	}
+
+	log := ctrl.LoggerFrom(ctx)
+	// TODO: improve formatting; diff is hard to read with all the escape chars
```

Comment:
I'm not sure how we could improve the output. Maybe it's already good as is because I guess in tools like Kibana it should be possible to print a nice multi-line output based on our escape characters. 

I'm personally fine with just copy&pasting the log line + sed in cases I have to debug the diff.

Reply:
I agree, we never know how many changes can be in diff. It's important that this log is present, processing it can be left to the user.

Reply:
Removed until we came up with a better cmp logic. 
We still have patches (which gives an hint of the diff, even if at the end of processing)